Task3/Task3_Ivaylo_Keremidarski/src/car_wash/python/utils/user.py

```diff
@@ -0,0 +1,27 @@
+import argparse
+
+
+parser = argparse.ArgumentParser()
+
+parser.add_argument("--ro", type=int, action="store_true")
```

Comment:
parser should be in main.py (main method). Please add default value and description.